### PR TITLE
Add ProfilePage, feat: user info and loan list

### LINKED ISSUE #1: [frontend] profile page component
We need to complete the specific UI of the profile page.

### PR DESCRIPTION
**Pull request by ghost**
resolved #1 

* unstyled (eg. Modal window for details)
* not including review list

### CODE REVIEW

File: backend/account/views.py [hunk 1]
```diff
@@ -37,9 +37,10 @@ def profile(request, user_pk):
     """
     prof = get_object_or_404(Profile, pk=user_pk)
     if request.method == 'GET':
-        if not request.user.is_authenticated:
-            return HttpResponse(status=401)
+        # if not request.user.is_authenticated:
+        #    return HttpResponse(status=401)
         dict_profile = model_to_dict(prof)
+        dict_profile['username'] = prof.__str__()
```
Comment (nevivurn) [inline backend/account/views.py]: use `str(prof)` instead?

File: backend/account/views.py [hunk 2]
```diff
@@ -37,9 +37,10 @@ def profile(request, user_pk):
     """
     prof = get_object_or_404(Profile, pk=user_pk)
     if request.method == 'GET':
-        if not request.user.is_authenticated:
-            return HttpResponse(status=401)
+        # if not request.user.is_authenticated:
+        #    return HttpResponse(status=401)
```
Comment (nevivurn) [inline backend/account/views.py]: We should be checking if the user is indeed authenticated.

File: backend/loan/views.py [hunk 3]
```diff
@@ -16,8 +17,8 @@ def loan_list(request):
     if request.method != 'GET' and request.method != 'POST':
         return HttpResponseNotAllowed(['GET', 'POST'])
 
-    if not request.user.is_authenticated:
-        return HttpResponse(status=401)
+    # if not request.user.is_authenticated:
+    #    return HttpResponse(status=401)
```
Comment (nevivurn) [inline backend/loan/views.py]: Same here ^

File: backend/loan/views.py [hunk 4]
```diff
@@ -1,8 +1,9 @@
 import json
 import datetime
-#from django.shortcuts import render
+# from django.shortcuts import render
 from json import JSONDecodeError
 from django.http import HttpResponse, HttpResponseNotAllowed, JsonResponse
+from django.core import serializers
```
Comment (nevivurn) [inline backend/loan/views.py]: This doesn't seem to be used anywhere.

File: backend/loan/views.py (hunk 3 above)
Comment (nevivurn) [reply backend/loan/views.py]: This file seems to cause merge conflicts (and changes to it are unnecessary), so I would consider just reverting this file.

File: frontend/src/pages/ProfilePage/Profile/container.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+const Profile = () => {
+  const [isLoading, setLoading] = useState(true);
+  const [username, setUsername] = useState('');
+  const [userId, setUserID] = useState(-1);
+  const [kakaoId, setKakaoId] = useState('');
+  const [phone, setPhone] = useState('');
+  const [bio, setBio] = useState('');
+
+  const targetUrl = '/account/user/1';
```
Comment (nevivurn) [inline frontend/src/pages/ProfilePage/Profile/container.jsx]: Don't hardcode this, instead get the current userId somehow and pass it there instead. We need to implement #50 anyway, might as well go ahead with it.

File: frontend/webpack.config.js [hunk 6]
```diff
@@ -1,7 +1,8 @@
 module.exports = {
   devServer: {
     proxy: {
```
Comment (nevivurn) [inline frontend/webpack.config.js]: Turns out we don't need this file at all. Just go ahead and remove it.

**Review by nevivurn — CHANGES_REQUESTED** [review]

File: backend/account/views.py (hunk 1 above)
Comment (nungnung) [reply backend/account/views.py]: Thanks for your feedback! I will fix it.

File: backend/account/views.py (hunk 2 above)
Comment (nungnung) [reply backend/account/views.py]: Sharing login status is not yet implemented. I will remove comments later.

**Review by nevivurn — APPROVED** [review]

**Pull request merged**